front/src/common/redux/auth/auth.api.js

```diff
@@ -6,7 +6,7 @@ async function loginApi(request) {
     const headers = { headers: { 'Content-Type': 'application/json' } };
 
     return axios.post(url, data, headers)
-        .then(response => response)
+       .then(response => response)
```

Comment:
Please, return from API function only actual data, don't return whole response. Saga should not know anything about http related stuff

Reply:
done